<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>351</width>
    <height>275</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Menu Utama</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="btnDetailPO">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Detail PO</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QPushButton" name="btnPemasok">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Pemasok</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QPushButton" name="btnMaterial">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Material</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPushButton" name="btnPelanggan">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Pelanggan</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnPO">
        <property name="minimumSize">
         <size>
          <width>160</width>
          <height>48</height>
         </size>
        </property>
        <property name="text">
         <string>Purchase Order</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>351</width>
     <height>25</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
